core/src/client/client.rs

```diff
@@ -318,18 +318,6 @@ impl EngineInfo for Client {
                 .seq()
         })
     }
-
-    fn possible_authors(&self, block_number: Option<u64>) -> Result<Option<Vec<PlatformAddress>>, EngineError> {
```

Comment:
I just realized that not only the commit message, but the commit itself was a mistake.
SInce `possible_authors` just returns a validator set, we don't have to remove this.
I'll restore this.
Maybe we should change the type of `PlatformAddress` in the future.